package-lock.json

```diff
@@ -9288,9 +9288,9 @@
       "integrity": "sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c="
     },
     "typescript": {
-      "version": "3.8.0-dev.20191030",
-      "resolved": "https://registry.npmjs.org/typescript/-/typescript-3.8.0-dev.20191030.tgz",
-      "integrity": "sha512-H4ICJQz3KmUy1V7MyBK+YjTDNTHPGuSIbCYxgJbhn/klrgs2P3jBuuS9neb5hpRNe4WU+dtXzPbfrl19czJgiw=="
+      "version": "3.8.0-dev.20191212",
+      "resolved": "https://registry.npmjs.org/typescript/-/typescript-3.8.0-dev.20191212.tgz",
+      "integrity": "sha512-fIxJcBYMqhuep+fYNBmCYT9HjAYS5Dajdgr33p/v+m4DdpOIRh0TJ9NmT0e3NB0Bq1VB12sOi8P/VMPIbWNmGw=="
     },
     "uglify-js": {
       "version": "3.6.4",
```

Comment:
@orta FYI I found that as the typescript package’s version requirement is set to `next`, you need to run `npm ci` to get exactly the version pinned in the lockfile.